src/SprykerEco/Service/CrefoPay/CrefoPayConfig.php

```diff
@@ -25,4 +36,34 @@ public function getCrefoPayOrderIdLength(): int
     {
         return static::CREFO_PAY_ORDER_ID_LENGTH;
     }
+
+    /**
+     * @api
+     *
+     * @return int
+     */
+    public function getCrefoPayUserIdMaxLength(): int
+    {
+        return static::CREFO_PAY_USER_ID_MAX_LENGTH;
+    }
+
+    /**
+     * @api
+     *
+     * @return int
+     */
+    public function getCrefoPayBasketItemIdMaxLength(): int
+    {
+        return static::CREFO_PAY_BASKET_ITEM_ID_MAX_LENGTH;
+    }
+
+    /**
+     * @api
+     *
+     * @return bool
+     */
+    public function getIsBusinessToBusiness(): bool
```

Comment:
```suggestion
    public function isBusinessToBusiness(): bool
```

Please don't use `get*` for methods with bool return value. This should imho something like isBusinessToBusiness or hasBusinessToBusiness.

Reply:
This was called that way since we have similar method in Zed config.